src/Microsoft.Azure.WebJobs/ParameterBindingData.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// A reference type for supporting SDK-type bindings in out-of-process Function workers
```

Comment:
nit: Id prefer if this doc wasn't functions/scenario specific, since the SDK is general and not for Functions only.

How about something like "Represents data a parameter binding provides that can be used by a function to perform the bind operation to the target itself."